cmd/sync_assets_with_s3.go

```diff
@@ -0,0 +1,99 @@
+// Copyright © 2016 Shingo Omura <everpeace@gmail.com>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"log"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+//
+// Flags
+//
+var bucketRegion string
+var bucketName string
+
+//
+// commands
+//
+var assetFileNames = []string{
+	"cluster.yml",
+	"terraform.tfstate",
+	"host_key",
+	"host_key.pub",
+	"worker_key",
+	"worker_key.pub",
+	"session_signing_key",
+	"session_signing_key.pub",
+	"authorized_worker_keys",
+}
+
+// putStatesToS3 represents the up command
+var putStatesToS3 = &cobra.Command{
+	Use:   "put-states-to-s3",
```

Comment:
More shorter names like `save` instead of `put-states-to-s3`, `restore` instead of `get-states-from-s3` would be fine because those shorter names would describe intents enough.